### PR TITLE
Allow passing aerospike credentials via environment variables

### PR DESCRIPTION
**Pull request by MaheshGPai**
Provision to allow aerospike credentials to be supplied though environment variables

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -64,6 +64,13 @@ func main() {
 		log.Fatal("usage error")
 	}
 
+	if(len(os.Getenv("AS_USER"))) == 0 {
```
Comment (alicebob) [inline main.go]: I think you have the logic the wrong way around :)

How about:

```
if user := os.Getenv("AS_USER"); user != "" {
  *username = user
}
```
that way you won't repeat the `AS_USER` string.

Comment (alicebob) [reply main.go]: the flag is called `-username`, so maybe it's less confusing to call the variable `AS_USERNAME`. That also matches nice with `AS_PASSWORD` (versus `AS_USER` and `AS_PASS`)...

Comment (MaheshGPai) [reply main.go]: My bad. Updated the commit

**Comment by alicebob** [comment]
Thanks!

**Pull request merged**